guide/foundations/principles.md

```diff
@@ -0,0 +1,182 @@
+
+Security can also be a feeling. A product that is polished, good looking, easy to use, and transparently communicates how it works can help users feel more secure– especially when compared to another product with the exact same security measures, but lacks these qualities.
+
+
+**Do**
+- Take safeguarding of users' funds seriously
+- Strive for no loss of funds, whether by negligence or theft 
+- Provide suitable private key management schemes for beginners
+- Offer progressive security and upgrade paths
+- Build with bad actors in mind
+- Minimize risk of self-inflicted loss from user negligence
+- Continuously educate users on best practice and risks 
+- Reduce attack surfaces by minimizing use of external code dependencies
+
+**Don't**
+- Blame the user for losing funds
+- Expect beginners to implement best practice backup strategies
+- Underestimate the added *feeling* of security that can come from well polished products
+
+---
+
+## Inclusion
+
+> Build borderless products without location, language or social barriers
+
+There are no background checks, credit checks, or gatekeepers to Bitcoin. A Kenyan farmer has the same access to Bitcoin as a Wall Street trader. 
+
+While Bitcoin is already used by a large number of people, it pales in comparison with the many more that are likely to use it in the future. We need to design products that are prepared for people unfamiliar with Bitcoin. This means using plain and familiar language, explaining things in the context where they are needed, not overwhelming people with technical detail, and more.
+
+**Do**
+- Provide equal and direct access to the Bitcoin network
+- Design Bitcoin products that are usable by the widest range of people possible
+- Use plain language that people new to Bitcoin can understand regardless of prior knowledge
+- Localize your product and make it multilingual 
+- Educate in place, when people are presented with a new concept
+- Treat users who rely on assistive technologies as first-class citizens
+
+**Don't**
+- Exclude people by building features that only work in certain countries
+- Add technical detail that is not required knowledge, or technical terms like seed phrase, XPUBs, mnemonics etc.
+- Put all education up front and expect people to read and remember it
+
+---
+
+## Interoperability
+
+> Enable import and export of wallets, maximise backwards compatibility and use of open standards
+
+Bitcoin is an open-source protocol, operated in a decentralized manner. This has led to a number of standards being developed to ensure compatibility between products. It should be easy to switch and move your Bitcoin wallet to a different application, should you wish. Making sure that your product supports as many of these standards as possible is best practice and builds trust. More on [wallet interoperability]({{ '/guide/foundations/wallet-interoperability/' | relative_url }}). 
```

Comment:
```suggestion
Bitcoin is an open-source protocol operating in a decentralized manner. This has led to a number of standards being developed to ensure compatibility between products. It should be easy to switch and move your Bitcoin wallet to a different application, should you wish. Ensuring that your product supports as many of these standards as possible is best practice and builds trust—more on wallet interoperability.
```

Reply:
Will decline this change as it removes the link to wallet-interoperability. But will apply the other changes through new commit.

Reply:
My bad completely forgot about the link, I've now included it if you haven't already added it. Nice catch.